core/kazoo_data/src/kz_datamgr.erl

```diff
@@ -110,7 +110,7 @@
 %%--------------------------------------------------------------------
 -spec update_doc_from_file(ne_binary(), atom(), nonempty_string() | ne_binary()) ->
                                   {'ok', kz_json:object()} |
-                                  data_error().
+                                  kazoo_data:data_error().
```

Comment:
@jamesaimonetti i think this was previously discussed. `data_error()` is part of the library (local type). prepending `kazoo_data:` gives you the feeling of an external type. thoughts ?